01_P/P_2_1_1_02/sketch.js

```diff
@@ -60,10 +61,8 @@ function setup() {
 
 function draw() {
   background(255);
-  smooth();
```

Comment:
Removed smooth() since this is already enabled by default in every p5 sketch (as well as Processing 2 and 3).